tests/unit/test_sagemaker_steps.py

```diff
@@ -156,6 +191,31 @@ def pca_model():
     )
 
 
+@pytest.fixture
+def pca_model_with_env():
```

Comment:
I've added a test using this Model that has defined env, vpc_config and imagie_config

Reply:
If this is important, write it in a code comment